specification/src/main/asciidoc/platform/RevisionHistory.adoc

```diff
@@ -7,10 +7,10 @@
 
 * Java EE 8 requires Java SE 8.
 * Added
```

Comment:
The revision history should be cleared out and replaced for Jakarta EE 8.

Reply:
I the Revision History does show the history before Jakarta...  I'd rather leave it.

Reply:
Didn't we remove all the acknowledgements for those versions?

As it is, the revision history is only for the lifetime of the Java EE 8 spec.
We clear out the revision history with each new version.  Seems like we
should do the same here, indicating how this version of this Jakarta EE
spec came to be.  The things that happened early in the Java EE 8 spec
development are irrelevant.